tests/RobotFramework/README.md

```diff
@@ -49,12 +49,12 @@ To run an robot file ssh to 192.168.1.4 and use the following command structure:
 ### 4.1. Task Automations
 ### **Installing specific SW Build**
 
-	File name: `build_install_rpi.robot`	
+	File name: `build_install_rpi.robot`
```

Comment:
This whole section is not rendering properly in markdown, it is being rendered as a code block instead of a list